README.md

```diff
@@ -40,7 +40,7 @@ INCLUDE_DIRECTORIES(${SDL2_INCLUDE_DIRS} ${SDL2IMAGE_INCLUDE_DIRS})
 TARGET_LINK_LIBRARIES(${PROJECT_NAME} ${SDL2_LIBRARIES} ${SDL2IMAGE_LIBRARIES})
 ```
 
-die Datei sollte etwa so aussehen: 
```

Comment:
Da hat mein Editor doch glatt noch einen überflüssigen Whitespace entfernt 😅 
Sollte eigentlich nicht unbedingt in diesem Commit sein, ich denke aber es sollte nicht schaden und bedarf keiner Änderung.